controllers/vdoconfig_controller.go

```diff
@@ -122,6 +119,13 @@ func (r *VDOConfigReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 		Logger:  r.Logger,
 	}
 
+	// Get VDO Namespace
+	VDO_NAMESPACE = os.Getenv("VDO_NAMESPACE")
+
+	if VDO_NAMESPACE == "" {
+		return ctrl.Result{}, errors.New("No Valid namespace found for VDO")
```

Comment:
lets word as unable to determine operator namespace

Reply:
Thanks Rahav, I have updated the error message as suggested.